cmd/opm/index/export.go

```diff
@@ -40,7 +39,7 @@ func newIndexExportCmd() *cobra.Command {
 	if err := indexCmd.MarkFlagRequired("index"); err != nil {
 		logrus.Panic("Failed to set required `index` flag for `index export`")
 	}
-	indexCmd.Flags().StringP("package", "o", "", "the package to export")
+	indexCmd.Flags().StringP("package", "p", "", "the package to export")
```

Comment:
Can we make this an alias? If users are already using `-o` this will break them.

Reply:
@kevinrizza Thanks for your review! I'm sorry, but I'm not familiar with the [cobra](https://github.com/spf13/cobra), do you have any suggestion to add the `alias` for a flag? Something like below:
```console
Flags:
  -p, -o --package string           the package to export
```

Reply:
@jianzhangbjz If you're unable to find a way to alias this, I suggest removing this client facing API change.

Reply:
@njhale Ok, good idea, thanks!